pkg/index/types.go

```diff
@@ -58,3 +58,8 @@ type FileOperation struct {
 	From string `json:"from,omitempty"`
 	To   string `json:"to,omitempty"`
 }
+
+// Receipt describes a plugin receipt file.
```

Comment:
`plugin receipt` &rarr; `plugin installation receipt`.

Honestly we can call the type `InstallReceipt` as well. @corneliusweig thoughts?

Reply:
Yeah it's fine + we can refactor easily.